src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -471,6 +472,17 @@ private IEnumerable<(ManifestId id, ManifestVersion version, SdkFeatureBand feat
                     ManifestVersion manifestVersion;
                     SdkFeatureBand manifestFeatureBand;
                     var parts = manifest.Value.Split('/');
+                    
+                    string manifestVersionString = (parts[0]).ToString();
```

Comment:
Is `ToString` needed? Split should be returning a string[]

Reply:
Yeah, it's not needed I don't know why I had it there. Thanks!